scripts/colmap2nerf.py

```diff
@@ -389,3 +390,40 @@ def closest_point_2_lines(oa, da, ob, db): # returns point closest to both rays
 	print(f"writing {OUT_PATH}")
 	with open(OUT_PATH, "w") as outfile:
 		json.dump(out, outfile, indent=2)
+
+	if len(args.mask_categories) > 1:
```

Comment:
This seems to require at least _two_ categories to be present. Is that intentional?

Reply:
This is just a dumb bug, fixed!